passes/opt/opt_merge.cc

```diff
@@ -140,8 +140,13 @@ struct OptMergeWorker
 			hash_conn_strings.push_back(s + "\n");
 		}
 
-		for (auto &it : cell->parameters)
-			hash_conn_strings.push_back("P " + it.first.str() + "=" + it.second.as_string() + "\n");
```

Comment:
I don't understand why we want this change. The original is more readable

Reply:
This is a functional change that hashes fewer characters. However, since we're not concerned with string parameters, only string attributes, and this only works on parameters, the change wouldn't be used, so I guess I'll put it back as you suggest